tests/integration/components/polaris-empty-state-test.js

```diff
@@ -1,254 +1,232 @@
-import { moduleForComponent, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import { find, click } from 'ember-native-dom-helpers';
```

Comment:
let's use the new `find`/`click` helpers provided by `@ember/test-helpers` instead of native-dom ones, I think only difference is you'll want `await click...`